jobs/ci-kubernetes-node-kubelet-conformance.sh

```diff
@@ -19,4 +19,23 @@ set -o pipefail
 set -o xtrace
 
 # PROJECT="k8s-jkns-ci-node-e2e"
-test/e2e_node/jenkins/conformance/conformance-jenkins.sh test/e2e_node/jenkins/conformance/jenkins-conformance.properties
```

Comment:
I think it might be best to test the conformance test separately before converting it. I'm worried that this isn't going to work as-is.